site/guides/namespace-provisioning-ui.md

```diff
@@ -0,0 +1,184 @@
+# Namespace provisioning example using UI
+
+In this guide, we will use the package orchestration UI to:
+
+- Register blueprint and deployment repositories
+- Create a `kpt` package from scratch in blueprint repo
+- Create a deployable instance of the `kpt` package
```

Comment:
do we have a doc to talk about "variant constructor"?  It seems confusing especially for existing kpt users.